src/meta/src/manager/catalog/fragment.rs

```diff
@@ -173,7 +173,7 @@ pub struct FragmentManager {
     core: RwLock<FragmentManagerCore>,
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct InflightFragmentInfo {
```

Comment:
Could we add some docs about the lifetime and usage of this struct?

Like when it is constructed, what it is used for, when it is dropped?

Reply:
The `InflightFragmentInfo` is just a plain info rather than some states, so it does not have any lifetime transition. Do you mean `InflightActorState`?